modal/cli/app.py

```diff
@@ -14,6 +14,7 @@
 from modal.environments import ensure_env
 from modal.exception import deprecation_warning
 from modal.object import _get_environment_name
+from modal.token_flow import _open_url
```

Comment:
I guess it would be better for this to live somewhere more expected if we're going to share it across multiple places but we can just move it later.